l5-operator/Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.0.1
+VERSION ?= 0.0.3
```

Comment:
apparently this should always be 0.0.0 for the dev branch per discussion in slack.. ok for now but we can revisit

Reply:
I imagine we will reset this number to 0.0.1 ( default ) for now, it's fine.